playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -15,7 +15,11 @@
 # Variables set to "None" will be converted to None
 # when the edxapp config is written to disk.
 
-EDXAPP_TIME_ZONE_DISPLAYED_FOR_DEADLINES: 'UTC'
+EDXAPP_TIME_ZONE_DISPLAYED_FOR_DEADLINES: 'US/Pacific'
+
+LANGUAGES:
+  - ['en', 'English']
+  - ['es-419', 'Latin-American Spanish']
```

Comment:
make this `"Espa\u00f1ol"` also rename dict to `EDXAPP_LANGUAGES` and map into `generic_env_config`